File: app/src/main/java/bj/rxjavaexperimentation/model/listing/Seller.java

package bj.rxjavaexperimentation.model.listing;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Seller {

    @SerializedName("username")
    @Expose
    private String username;
    @SerializedName("resource_url")
    @Expose
    private String resourceUrl;
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("shipping")
    @Expose
    private String shipping;
    @SerializedName("payment")
    @Expose
    private String payment;

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getResourceUrl() {
        return resourceUrl;
    }

    public void setResourceUrl(String resourceUrl) {
        this.resourceUrl = resourceUrl;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getShipping() {
        return shipping;
    }

    public void setShipping(String shipping) {
        this.shipping = shipping;
    }

    public String getPayment() {
        return payment;
    }

    public void setPayment(String payment) {
        this.payment = payment;
    }

}

File: app/src/main/java/bj/vinylbrowser/model/common/Seller.kt
package bj.vinylbrowser.model.common

import android.os.Parcel
import android.os.Parcelable
import com.google.gson.annotations.SerializedName

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class Seller(var username: String = "",
                  @SerializedName("resource_url") var resourceUrl: String = "",
                  var id: String = "",
                  var shipping: String = "",
                  var payment: String = "") : Parcelable {

    override fun writeToParcel(dest: Parcel, flags: Int) {
        dest.writeString(username)
        dest.writeString(resourceUrl)
        dest.writeString(id)
        dest.writeString(shipping)
        dest.writeString(payment)
    }

    override fun describeContents(): Int {
        return 0
    }
}